pkg/debug/spec.sh

```diff
@@ -260,6 +262,7 @@ for VGA in $(lspci -D  | grep VGA | cut -f1 -d\ ); do
         "PciLong": "${VGA}"
       },
       "logicallabel": "VGA${ID}",
+      "description": "${desc}",
```

Comment:
I just realized that the commercial controller will reject unknown fields (like description) in the json. That's why they are currently only when verbose.
We should add a description field to the controller, but in the mean time let's only output the decription when -v is specified.